extensions/dacpac/src/wizard/dataTierApplicationWizard.ts

```diff
@@ -119,6 +124,16 @@ export class DataTierApplicationWizard {
 		this.configureButtons();
 
 		this.wizard.open();
+
+		// Data tier wizard cancel button click event
+		this.wizard.cancelButton.onClick(() => {
+			TelemetryReporter.createActionEvent(TelemetryViews.DataTierApplicationWizard, 'DataTierApplicationWizardClosed')
+				.withAdditionalProperties({
+					'dacpacWizardClosedPage': this.currentPage
+				}).send();
+		});
+		//Reporting Dacpac wizard successful event to Telemetry
+		TelemetryReporter.sendActionEvent(TelemetryViews.DataTierApplicationWizard, 'DataTierApplicationWizardLaunched');
```

Comment:
This shouldn't be necessary - just pass in 'DataTierApplicationWizard' as the name to the createWizard call on line 90 and it should generate a `ModalDialogOpened` action event with that name already when the wizard is opened.

Reply:
👍